draft-ietf-quic-transport.md

```diff
@@ -4341,36 +4334,40 @@ Stream ID:
 
 : A variable-length integer indicating the stream which is flow control blocked.
 
-Offset:
+Data Limit:
 
 : A variable-length integer indicating the offset of the stream at which the
   blocking occurred.
 
 
-## STREAM_ID_BLOCKED Frame {#frame-stream-id-blocked}
+## STREAMS_BLOCKED Frame {#frame-streams-blocked}
 
-A sender SHOULD send a STREAM_ID_BLOCKED frame (type=0x0a) when it wishes to
-open a stream, but is unable to due to the maximum stream ID limit set by its
-peer (see {{frame-max-stream-id}}).  This does not open the stream, but informs
-the peer that a new stream was needed, but the stream limit prevented the
-creation of the stream.
+A sender SHOULD send a STREAMS_BLOCKED frame (type=0x1e or 0x1f) when it wishes
+to open a stream, but is unable to due to the maximum stream ID limit set by its
+peer (see {{frame-max-streams}}).  A STREAMS_BLOCKED frame of type 0x1e is used
+to indicate reaching the bidirectional stream limit; a STREAMS_BLOCKED frame of
+type 0x1f indicates reaching the unidirectional stream limit.
```

Comment:
Again, flag might be cleaner.

Reply:
Similarly, UNI_STREAMS_BLOCKED and BIDI_STREAMS_BLOCKED.

Reply:
Drop this suggestion too.